elasticpress-woocommerce.php

```diff
@@ -265,10 +280,8 @@ function epwc_translate_args( $query ) {
 			$query->set( 'meta_query', $meta_query );
 		}
 
-		$post_parent = $query->get( 'post_parent', false );
-
-
-		if ( 'product' === $post_type ) {
+		// Assuming empty to be product post type
+		if ( empty( $post_type ) ) {
```

Comment:
I have changed that line to include the product post type checking too. The check for empty is added as this is empty on the product cat archives and we do not want the meta query to execute on all the remaining WooCommerce post type queries.